src/GovUk.Education.ExploreEducationStatistics.Content.Services/ViewModels/PublicationViewModel.cs

```diff
@@ -40,11 +42,12 @@ public PublicationViewModel()
         Slug = publication.Slug;
         LatestReleaseId = publication.LatestReleaseId;
         IsSuperseded = publication.IsSuperseded;
+        SupersededBy = publication.SupersededBy!;
```

Comment:
You won't need the null-forgiving operator `!` here when you make this optional. That was a warning that something was not quite right here.